app/src/main/res/values/styles_v2.xml

```diff
@@ -328,6 +328,24 @@
     <item name="android:drawablePadding">8dp</item>
   </style>
 
+  <style name="Clinic.V2.Button.WithIcon.Green3">
+    <item name="android:background">@drawable/background_button_solid_green3</item>
+    <item name="android:foreground">?selectableItemBackground</item>
+    <item name="android:textColor">@color/green1</item>
+    <item name="android:paddingTop">0dp</item>
+    <item name="android:paddingBottom">0dp</item>
+    <item name="android:drawablePadding">8dp</item>
+  </style>
+
+  <style name="Clinic.V2.Button.WithIcon.Green1">
+    <item name="android:background">@drawable/background_button_solid_green1</item>
+    <item name="android:foreground">?selectableItemBackground</item>
+    <item name="android:textColor">@color/white</item>
```

Comment:
Can you extract this into a separate `TextAppearance`?

Reply:
done